src/exercise/05.js

```diff
@@ -12,16 +12,32 @@ import '../box-styles.css'
 
 // 🐨 add a style prop to each of them as well so their background color
 // matches what the text says it should be as well as `fontStyle: 'italic'`
-const smallBox = <div>small lightblue box</div>
-const mediumBox = <div>medium pink box</div>
-const largeBox = <div>large orange box</div>
+// const smallBox = <div className="box box--small" style={{backgroundColor: 'lightblue', fontStyle: 'italic'}}>small lightblue box</div>
+// const mediumBox = <div className="box box--medium" style={{backgroundColor: 'pink', fontStyle: 'italic'}}>medium pink box</div>
+// const largeBox = <div className="box box--large" style={{backgroundColor: 'orange', fontStyle: 'italic'}}>large orange box</div>
+
+const Box = ({size, style = '', children}) => {
+  return (
+    <div
+      className={`box box--${size}`}
+      style={{fontStyle: 'italic', ...style}}>
+        {children}
+    </div>
+  );
+}
 
 function App() {
   return (
     <div>
-      {smallBox}
-      {mediumBox}
-      {largeBox}
+      <Box size="small" style={{backgroundColor: 'lightblue'}}>
+        small lightblue box
+      </Box>
+      <Box size="medium" style={{backgroundColor: 'pink'}}>
+        medium pink box
+      </Box>
+      <Box size="large" style={{backgroundColor: 'orange'}}>
+        large orange box
+      </Box>
```

Comment:
Try adding a `<Box>no size</Box>` here with no size attribute to see what happens. In your function above, you could check if a size is being passed before rendering it. (Note this isn't clear in the instructions, but you'll see it in the answer key - 05.extra-2.js)